components/services/bat_ledger/public/interfaces/bat_ledger.mojom

```diff
@@ -76,7 +76,7 @@ interface BatLedger {
   OnTimer(uint32 timer_id);
 
   GetBalanceReport(ledger.mojom.ActivityMonth month, int32 year) =>
-      (ledger.mojom.Result result, ledger.mojom.BalanceReportInfo report);
+      (ledger.mojom.Result result, ledger.mojom.BalanceReportInfo? report);
```

Comment:
This change fixes a debug crash which occurs `GetBalanceReport` is called and an error condition is encountered.